public/content/lessons/2019/windmills/index.mdx

```diff
@@ -1,9 +1,232 @@
+
+<Figure
+    image="change_pivot.png"
+    video="change_pivot.mp4"
+/>
+
+Great, that’s insight #1. Now why would this imply that the line must hit every point infinitely many times, no matter what weird set of points you dream up? Well, think about letting this process go until the line has turned 180 degrees. That means it’s parallel to its starting position, and because it has to remain the case that half the points are on one side, half are on the other, it must be passing through the same point it started on. Otherwise, if it ended up on some other point, it would change the number on a given side. Additionally, since the line has rotated halfway around, everything that was blue has become brown, and everything which was brown has become blue, and the only way to change color is if you get hit by the line.
+
+<Figure
+    image="odd_img2.png"
+    video="odd_img.mp4"
+    show="video"
+/>
+
+So for our odd number case, that means that after a half rotation, the line is back to where it started, having hit all the other points. So as time moves forward, it repeats that exact same set of motions over and over, hitting all points infinitely many times.
+
+If there are an even number of points, we need to alter the scheme slightly, but only slightly. Let’s say that in that case, the pivot counts as a brown point. So we can still select an initial point so that half the points are blue, all on the left, and half are brown, now either meaning their either on the right or the pivot.
+
+<Figure
+    image="even_intro.png"
+/>
+
+The same argument applies that after a 180-degree turn, all the points will have swapped color, although this time the line might be passing through a different point after that first half turn, specifically one that used to be blue. But after another 180 degrees the line must be passing through the same point it started on. Why? Again, it must be parallel to its starting position, and if it was passing through any other point, the number of points on a given side would be different.
+
+<Figure
+    image="even_img.png"
+    video="even_img.mp4"
+    show="video"
+/>
+
+So again, you have a cycle which hits all points, and which ends in the same position where it started, so it must hit all points infinitely many times.
+
```

Comment:
Below is something we could insert here. Maybe as an accordion?

---

From the comments on the original video for this topic, I learned that many people didn't quite realize that the problem had been solved at this point. Here's what I wrote in response:

> A few people ask "but what is the formal solution?".  Take a look at [this writeup](https://artofproblemsolving.com/wiki/index.php?title=2011_IMO_Problems/Problem_2), which you'll find presents an argument with the same essential idea as was presented here.  It would get you full marks on the test: 
> 
> The fact that an argument doesn't reference equations and sets doesn't necessarily make it incomplete.  Sure, you could explicitly define what words like "left" and "right" refer to by defining such and such cross product with such and such parameterization of the line, but for most readers, it wouldn't actually remove any ambiguity.  The purpose of formality is to make all the terms used unambiguous to all readers, not to dress up the language to involve sufficiently many symbols. If you understood this video, you understood the full solution.